test/parallel/test-error-codes-impl-doc.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+// Flags: --expose-internals
+
+// Note: this script reads Node.js sources and docs, and expects to be run on
+// the very same Node.js version as the source and doc tree is.
+
+require('../common');
+const assert = require('assert');
+const errors = require('internal/errors').codes;
+const fs = require('fs');
+const path = require('path');
+const natives = process.binding('natives');
+
+// --report prints only the list of failed checks and does not assert
+const report = process.argv[2] === '--report';
```

Comment:
I suggest to change that to always asserting but checking all entries first instead of asserting on the first failure.

That way the failures are obvious from the assertion and the user does not have to run the test with this flag (and it's not that obvious that this flag must be used by just taking a glimpse at this).